### PR TITLE
add placeholder image and use it

### PR DESCRIPTION
**Pull request by bishoymelek**
- Add placeholder Image to `/static` to be used instead of an API Placeholder provider.

- Add styling prop for the Image tag in `chooseEditImage` Atom and start using the placeholder image.

- Use the atom in `EditPeerPanel` with the styling prop and new placeholder image

### CODE REVIEW

File: packages/editor/src/client/panel/peerEditPanel.tsx [hunk 1]
```diff
@@ -191,16 +192,14 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
...
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
+            <ChooseEditImage
+              disabled
+              imageStyle={{
+                display: 'block',
+                marginLeft: 'auto',
+                marginRight: 'auto'
               }}
+              image={profile?.logo?.previewURL ?? '/static/placeholder-240x240.png'}
```
Comment (roosnic1) [inline packages/editor/src/client/panel/peerEditPanel.tsx:202]: is the `?? '/static/...'` necessary? you check for a valid url inside the component.

File: packages/editor/src/client/panel/peerEditPanel.tsx [hunk 2]
```diff
@@ -191,16 +192,14 @@ export function PeerEditPanel({id, onClose, onSave}: ImageEditPanelProps) {
...
-              style={{
-                height: '200px',
-                marginBottom: '20px',
-                backgroundSize: 'cover',
-                backgroundImage: `url(${
-                  profile?.logo?.previewURL ?? 'https://via.placeholder.com/240x240'
-                }`
+            <ChooseEditImage
+              disabled
+              imageStyle={{
```
Comment (roosnic1) [inline packages/editor/src/client/panel/peerEditPanel.tsx:197]: I think you can set them inside the component and check a few places where the component is used. I think it's needed everywhere.

**Review by roosnic1 — CHANGES_REQUESTED** [review]
looks good. But you can add the styles for the component in general.

File: packages/editor/src/client/panel/peerEditPanel.tsx (hunk 2 above)
Comment (bishoymelek) [reply packages/editor/src/client/panel/peerEditPanel.tsx:197]: I did that at the beginning but I am not sure about it. I will go through other places as well to see it

File: packages/editor/src/client/panel/peerEditPanel.tsx (hunk 1 above)
Comment (bishoymelek) [reply packages/editor/src/client/panel/peerEditPanel.tsx:202]: I am sorry I don't get what you mean

**Review by 0815pascal — DISMISSED** [review]

**Review by roosnic1 — CHANGES_REQUESTED** [review]
Tested this locally with the demo instance and got the following:
image

Did you test this?

**Review by roosnic1 — APPROVED** [review]
👍🏾

**Review by 0815pascal — APPROVED** [review]

**Review by penk85 — APPROVED** [review]

**Pull request merged**